diff --git a/config.def.h b/config.def.h
index defa426..d64d2ee 100644
--- a/config.def.h
+++ b/config.def.h
@@ -39,6 +39,9 @@ static Key keys[] = {
 	{ MODKEY|ShiftMask,     XK_Return, focusonce,   { 0 } },
 	{ MODKEY|ShiftMask,     XK_Return, spawn,       { 0 } },
 
+	{ MODKEY,               XK_Page_Up,   rotate,      { .i = -1 } },
+	{ MODKEY,               XK_Page_Down, rotate,      { .i = +1 } },
+
 	{ MODKEY|ShiftMask,     XK_l,      rotate,      { .i = +1 } },
 	{ MODKEY|ShiftMask,     XK_h,      rotate,      { .i = -1 } },
 	{ MODKEY|ShiftMask,     XK_j,      movetab,     { .i = -1 } },
